### PR TITLE
Add a specific message for stupid users that put useless spaces in license

### PR DESCRIPTION
**Pull request by RyDroid**
I was one of this stupid users, so I would like to help future ones.

### CODE REVIEW

File: src/Composer/Util/ConfigValidator.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
  *
  * (c) Nils Adermann <naderman@naderman.de>
  *     Jordi Boggiano <j.boggiano@seld.be>
+ *     Nicola Spanti <dev@nicola-spanti.info>
```
Comment (staabm) [inline src/Composer/Util/ConfigValidator.php]: not your software ;)

Comment (RyDroid) [inline src/Composer/Util/ConfigValidator.php]: Of course. Is it just a joke or do you want to have copyright for you ?

Comment (staabm) [inline src/Composer/Util/ConfigValidator.php]: Nope, but you shouldnt add your name their

Comment (RyDroid) [inline src/Composer/Util/ConfigValidator.php]: Ok, I removed it.

**Pull request merged**